docs/API.md

```diff
@@ -486,6 +538,22 @@ await expect(list).toHaveChildren(3) // the list has 3 items
 await expect(list).toHaveChildren({ eq: 3 })
 ```
 
+### toHaveSize
+
+Checks if element has a specific size.
+
+##### Usage
+
+```js
+await browser.url('http://github.com')
+const logo = await $('.octicon-mark-github')
+await expect(logo).toHaveSize({ width: 32, height: 32 })
+await expect(logo).toHaveSize(32, property: 'width')
+await expect(logo).toHaveSize(32, property: 'height')
```

Comment:
I suggest to rather have:

```ts
await expect(logo).toHaveSize({ width: 32, height: 32 })
await expect(logo).toHaveWith(32)
await expect(logo).toHaveHeight(32)
```

Also let's define what size means? The inner height of an element or the height including the margin etc

Reply:
I think that if this is unclear we should actually update the WDIO API first since we don't supply any info here either: https://webdriver.io/docs/api/element/getSize

Reply:
I like the idea splitting them up 👍

Reply:
> I think that if this is unclear we should actually update the WDIO API as well since we don't supply any info here either: https://webdriver.io/docs/api/element/getSize

You are right, I am totally open to that!